src/pages/Location2.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Location2() {
+  return (
+    <div className='loc2'>
+      <h1>Location 2</h1>
+    </div>
+  );
+}
+
+export default Location2;
```

Comment:
lgtm

Reply:
Looks good for now, I assume we want a more general way for requesting a list of locations instead of hardcoded ones right?